### PR TITLE
Implement proxy

### PR DESCRIPTION
**Pull request by trompx**
Hey @timgrossmann @Albertmoiseev,

As quickly discussed here (https://github.com/timgrossmann/InstaPy/pull/92), I thought it was about time to implement proxy.
I did this really quickly and did not have a setup to test it right away but it could be a good starting point to implement the feature.

Feel free to test and iterate. I'll try to test it tommorrow if I get some time.

Cheers

### CODE REVIEW

**Comment by timgrossmann** [comment]
@trompx Hey there, Awesome!

If you tested it, I'd be happy to merge this, thank you for Contributing!

File: quickstart.py [hunk 1]
```diff
@@ -11,6 +11,9 @@
 # username=<your-username-here> and password=<your-password> into InstaPy
 # e.g like so InstaPy(username="instagram", password="test1234")
 
+# If you want to use a proxy, use:
+# InstaPy(proxy="xx.xx.xx.xx:port")
```
Comment (timgrossmann) [inline quickstart.py:15]: I wouldn't put this in the quickstart file...
Please remove that

File: README.md [hunk 2]
```diff
@@ -95,6 +95,23 @@ session.like_from_image(url='www.instagram.com/image', amount=15, media='Video')
 session.end()
 ```
 
+##### Proxy
+
+If you want the script to get the proxy from your environment, you can do:
+
+```
```
Comment (timgrossmann) [inline README.md:109]: Could you also add that part to docs/index, please?

**Review by timgrossmann — CHANGES_REQUESTED** [review]

**Comment by timgrossmann** [comment]
@trompx Could you please fix and resolve?

**Comment by trompx** [comment]
Hello @timgrossmann,

Sorry I did not have anytime lately for this lately, I could fix it quickly but finally I don't feel like it is of a great use in that state (a proxy without auth could mean you use a public proxy which is really a bad idea and would increase chances of being banned).

I will try when I have a bit more time to implement the proxy but with authentification. Once I have something more solid and battle tested I'll make a new pull request.

**Comment by bardascat** [comment]
What is the status of this task? It's already older than 1 month...
Right now is it possible to run instapy for multiple instagram accounts without getting banned? 
If yes, how ?

**Comment by timgrossmann** [comment]
@bardascat Just clone InstaPy multiple times.

**Comment by bardascat** [comment]
@timgrossmann 
I know it's possible to clone InstaPy multiple time to use multiple instagram accounts.
In this case my concern is  if I login with multiple instagram accounts from the same ip I will get banned by instagram? Is this a real concern?
If it is what can I do, to avoid this?

Thanks

**Comment by timgrossmann** [comment]
@bardascat Haven't had any problems with it yet.

**Pull request closed without merging**